Sources/Commands/Options.swift

```diff
@@ -375,6 +375,9 @@ public struct SwiftToolOptions: ParsableArguments {
     @Option(name: .customLong("resolver-fingerprint-checking"))
     var resolverFingerprintCheckingMode: FingerprintCheckingMode = .warn
 
+    @Flag(name: .customLong("disable-dead-strip"), help: "Disable dead code stripping by the linker")
+    var disableDeadStrip: Bool = false
```

Comment:
```suggestion
    @Flag(name: .customLong("dead-strip"), inversion: .prefixedEnableDisable, help: "Disable/enable dead code stripping by the linker. Enabled by default")
    var enableDeadStrip: Bool = true
```

Reply:
Sounds good, updated, I figured since it was a disable only we didn't need the positive case, but nbd to me! Also it looks like swift-argument-parser automatically adds the default to the help so I removed that piece here